lib/elements/dom-bind.html

```diff
@@ -61,16 +61,19 @@
         this.__children = null;
       }
 
-      // assumes only one observed attribute
+      /** @returns {void} */
       attributeChangedCallback() {
+        // assumes only one observed attribute
         this.mutableData = true;
       }
 
+      /** @returns {void} */
```

Comment:
`@return` is only necessary if the function takes arguments

Reply:
See below. Think we should keep these.